apps/service_providers/tests/test_assistant_runnable.py

```diff
@@ -33,6 +33,7 @@ def session():
     return session
 
 
+@pytest.mark.django_db()
```

Comment:
Pity to add these since it slows down the tests quite a bit. What about patching the method?

Reply:
Sounds reasonable :+1: